great_expectations/expectations/expectation.py

```diff
@@ -2362,7 +2365,7 @@ class MulticolumnMapExpectation(BatchExpectation, ABC):
             the expectation.
     """  # noqa: E501
 
-    column_list: List[str]
+    column_list: List[StrictStr] = Field(min_items=1)
```

Comment:
I removed `Field(min_items=1)` because we don't know how this will always be implemented, and maybe an empty list is acceptable.

I left `StrictStr` in place of `str` because we shouldn't be allowing type coercion to `str` from other types for a column name. Column names that are numbers or come from some other serialized object don't make sense in practice.